tests/test_unit_tests.py

```diff
@@ -32,32 +32,34 @@ def test_version(self):
 class TestReader(unittest.TestCase):
     def test_stream_read(self):
         with open(testPath, "rb") as file:
-            reader = Reader("image/jpeg",file)
-            json = reader.json()
-            self.assertIn("C.jpg", json)
+            with Reader("image/jpeg", file) as reader:
```

Comment:
Changes to use context manager, so we show "best" rpactices (if not best, likely more idiomatic to Python).